tests/unit/Codeception/Command/BaseCommandRunner.php

```diff
@@ -54,6 +55,7 @@ protected function makeCommand($className, $saved = true)
                 $self->filename = $file;
                 $self->content = $output;
                 $self->log[] = ['filename' => $file, 'content' => $output];
+                $self->saved[$file] = $output;
```

Comment:
$self->log and $self->saved store the same data in different structure. Do we really need both?